core/textile/sync/pinning.go

```diff
@@ -55,6 +54,51 @@ func (s *synchronizer) uploadFileToRemote(ctx context.Context, bucket, path stri
 	return nil
 }
 
+func (s *synchronizer) downloadFileFromRemote(ctx context.Context, bucketSlug, path string) error {
```

Comment:
Maybe we can refactor `uploadFileToRemote` to use an auxiliary function used by both that method and this one instead of duplicating the function but swapping two vars

Reply:
@dmerrill6 

Thank you for this suggestion, made it more generic in https://github.com/FleekHQ/space-daemon/pull/207/commits/ef0d6f4f30a11598e5339bfa00e382e3df1774a7#diff-26096467a39167c453134f075440f43a014ed45f4479be47d6ba72aaa7976829R27